qa/suites/fs/verify/validater/valgrind.yaml

```diff
@@ -22,7 +22,6 @@ overrides:
       mon: [--tool=memcheck, --leak-check=full, --show-reachable=yes]
       osd: [--tool=memcheck]
       mds: [--tool=memcheck]
-      mgr: [--tool=memcheck]
```

Comment:
Commenting these out with a link to the tracker ticket would be better I think in case someone wants to try to fix it.